packages/lms/src/lib/components/content/Main.svelte

```diff
@@ -3,7 +3,7 @@
 </script>
 
 <main>
```

Comment:
```suggestion
<main class="p-3">
```
A bit of padding to move the text out of the edges

We can get rid of the <style> block (I can't select it)